lib/chef/provider/chef_organization.rb

```diff
@@ -52,7 +52,9 @@ def whyrun_supported?
     new_resource.members.each do |user|
       if !existing_members.include?(user)
         converge_by "Add #{user} to organization #{new_resource.name}" do
-          rest.post("#{rest.root_url}/organizations/#{new_resource.name}/users/#{user}", {})
```

Comment:
@poliva83 I'll take you through what I just did, so you can see the breadth of the code.  The start of our adventure lies here, in the creation of the rest object:

https://github.com/chef/cheffish/blob/master/lib/cheffish/chef_provider_base.rb#L8

Which runs through this class, with middleware for parsing the JSON output:

https://github.com/chef/cheffish/blob/master/lib/cheffish/server_api.rb#L44